index.html

```diff
@@ -4,11 +4,127 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<!-- <link rel="apple-touch-icon" sizes="120x120" href="/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/site.webmanifest">
+<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
+<meta name="msapplication-TileColor" content="#00aba9"> -->
+<link rel="shortcut icon" href="./images/favicon.png" type="image/x-icon">
+<meta name="theme-color" content="#3f9a72">
```

Comment:
Lines 7 - 14 should be indented, and you should remove your unused favicon configuration that you have commented out here unless you're planning on imminently making use of it.